staubli_tx90_support/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+cmake_minimum_required(VERSION 2.8.3)
+
+project(staubli_tx90_support)
+
+find_package(catkin REQUIRED)
+
+catkin_package()
+
+if (CATKIN_ENABLE_TESTING)
+  find_package(roslaunch REQUIRED)
+  roslaunch_add_file_check(tests/roslaunch_test.xml)
+endif()
+
+foreach(dir config launch meshes urdf)
+  install(
+    DIRECTORY ${dir}/
+    DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}/${dir})
+endforeach()
```

Comment:
The only reason why I implemented as such was for consistency with [ros-industrial/staubli/staubli_rx160_support/CMakeLists.txt] (https://github.com/ros-industrial/staubli/blob/hydro-devel/staubli_rx160_support/CMakeLists.txt).

I am happy to modify (both if needed, and make a PR on [ros-industrial/staubli](https://github.com/ros-industrial/staubli)).